knative-operator/pkg/controller/knativeserving/consoleclidownload/consoleclidownload.go

```diff
@@ -194,19 +194,19 @@ func populateKnConsoleCLIDownload(baseURL string, instance *servingv1alpha1.Knat
 			Description: "The OpenShift Serverless client `kn` is a CLI tool that allows you to fully manage OpenShift Serverless Serving and Eventing resources without writing a single line of YAML.",
 			Links: []consolev1.CLIDownloadLink{{
 				Text: "Download kn for Linux for x86_64",
-				Href: baseURL + "/amd64/linux/kn-linux-amd64.tar.gz",
+				Href: baseURL + "/kn-linux-amd64.tar.gz",
```

Comment:
For the backward compatibility, e.g. scripts relying on the URL format. Should we try to replicate the former format. Or it doesn't matter much since base is changing as well?

Reply:
We certainly could. You think it's worth it?

Reply:
Well, the URL was changed previously as well moving from `Deployment` to Ksvc. If I'm not mistaken there weren't any serious objections. I suppose it's fine as now as well.